README.md

```diff
@@ -40,23 +39,42 @@ cd theworld.org
 > By default, development environments will pull API data from the dev staging environemt. To pull data from the live API, in the `./config` directory, make a copy of `production.json` and rename it to `local.json`.
 
 Now we need to make sure we are using the the version of _node_ need for the app:
+
 ```
 nvm use
 ```
 
 Now lets install all the packages required by the app:
+
 ```
 yarn
 ```
 
+Next we need to setup the environment variables:
+
+```
+cp .env-example .env
+```
+
+Update variable values as needed. Some features, like newsletter subscriptions will not work until a valid key is provided here.
+
+If you need to use a local data API, do the following to create a local app config file:
+
+```
+cp ./config/local.example.js ./config/local.js
+```
+
+Update the settings here to point at you local dev API if they differ from the default.
+
 Finally, lets spin up the development server:
+
 ```
-now dev
+yarn dev:start
```

Comment:
Maybe add a step to confirm you are logged into Vercel? `yarn vercel login` before trying to start the app?